ferrocene/doc/sphinx-shared-resources/README.md

```diff
@@ -3,16 +3,15 @@
 
 # Ferrocene's shared Sphinx resources
 
-This repository contains the shared Sphinx resources we rely on for our
```

Comment:
This README.md could use a bit of words describing how users should bump the version number to publish a new version of the package. I know it's sorta in the `pyproject.toml`, but I don't think most folks would look there.

Reply:
This is the readme that will be published as a description on PyPI... If you want that, I can create a separate PyPI-specific readme.